<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>1024</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>PDU校准软件</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="readmeBtn">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>93</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>操作说明</string>
    </property>
   </widget>
   <widget class="QWidget" name="serialWid" native="true">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>10</y>
      <width>270</width>
      <height>60</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="titlelabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>PDU校准工具_V0.0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="modeWid" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>1261</width>
      <height>350</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="tableWid" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>1260</width>
      <height>591</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
